extensions/amp-date-picker/0.1/test/test-dates-list.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import '../../../../third_party/react-dates/bundle';
+import {DatesList} from '../dates-list';
+import {requireExternal} from '../../../../src/module';
+
+describes.sandboxed('DatesList', {}, () => {
+  const moment = requireExternal('moment');
```

Comment:
nit: seems more appropriate to be right after the `import` block

Reply:
I think it would be better inside the block, since it would mirror the style I use in the amp-date-picker classes. I prefer not to `requireExternal`  in the static context outside of a function, because the call could have side effects and I know #6089 was a change to avoid side-effect calls in static contexts